br/pkg/lightning/backend/kv/kv2sql.go

```diff
@@ -111,7 +105,6 @@ func (t *TableKVDecoder) IterRawIndexKeys(h kv.Handle, rawRow []byte, fn func([]
 }
 
 func NewTableKVDecoder(tbl table.Table, tableName string, options *SessionOptions) (*TableKVDecoder, error) {
-	metric.KvEncoderCounter.WithLabelValues("open").Inc()
```

Comment:
why delete

Reply:
`KvEncoderCounter` should record the mertics of `TableKVEncoder`.